angular/lib/src/lib/slider/slider.component.ts

```diff
@@ -0,0 +1,393 @@
+		widgetName: 'slider',
+		events: {
+			onValuesChange: (event) => {
+				this.emitControlValues();
+				this.valuesChange.emit(event);
+			},
+		},
+	});
+	/**
+	 * SliderWidget hold the state and actions applicable to the auSliderComponent
+	 */
+	readonly widget = toSlotContextWidget(this._widget);
+	readonly state: Signal<SliderState> = toSignal(this._widget.state$, {requireSync: true});
+	readonly api = this._widget.api;
+
+	/**
+	 * CSS classes to be applied on the widget main container
+	 */
+	@Input('auClassName') className: string | undefined;
+
+	/**
+	 * Minimum value that can be assigned to the slider
+	 */
+	@Input('auMin')
+	min: number | undefined;
+
+	/**
+	 * Maximum value that can be assigned to the slider
+	 */
+	@Input('auMax')
+	max: number | undefined;
+
+	/**
+	 * Unit value between slider steps
+	 */
+	@Input('auStepSize')
+	stepSize: number | undefined;
+
+	/**
+	 * Current slider values
+	 */
+	@Input('auValues')
+	values: number[] | undefined;
+
+	/**
+	 * If `true` slider value cannot be changed but the slider is still focusable
+	 */
+	@Input('auReadonly')
+	readonly: boolean | undefined;
+
+	/**
+	 * If `true` slider value cannot be changed and the slider cannot be focused
+	 */
+	@Input('auDisabled')
+	disabled: boolean | undefined;
+
+	/**
+	 * If `true` is vertically positioned otherwise it is horizontal
+	 */
+	@Input('auVertical')
+	vertical: boolean | undefined;
+
+	/**
+	 * An event emitted when slider values are changed
+	 *
+	 * Event payload equals to the updated slider values
+	 */
+	// TODO remove the disable when angular-check-props fixed
+	// eslint-disable-next-line @agnos-ui/angular-check-props
+	@Output('auValuesChange')
+	valuesChange = new EventEmitter<number[]>();
+
+	/**
+	 * Control value accessor methods
+	 */
+	onChange = (_: any) => {};
+
+	onTouched = () => {};
+
+	registerOnChange(fn: (value: any) => any): void {
+		this.onChange = fn;
+	}
+
+	registerOnTouched(fn: () => any): void {
+		this.onTouched = fn;
+	}
+
+	writeValue(value: any): void {
+		if (Array.isArray(value)) {
+			this._widget.patch({
+				values: value,
+			});
+		} else {
+			this._widget.patch({
+				values: [value],
+			});
+		}
+	}
+
+	setDisabledState(isDisabled: boolean) {
+		this._widget.patch({
+			disabled: isDisabled,
+		});
+	}
+
+	ngOnChanges(changes: SimpleChanges) {
+		patchSimpleChanges(this._widget.patch, changes);
+	}
+
+	ngAfterViewInit() {
+		this._sliderDirective = this._widget.directives.sliderDirective(this._elementRef.nativeElement);
+	}
+
+	ngOnDestroy() {
+		this._sliderDirective?.destroy?.();
+	}
+
+	handleBlur() {
+		this.onTouched();
+	}
+
+	/**
+	 * Method that holds all the necessary emits to update the input value of the slider
+	 */
+	emitControlValues() {
+		this.onChange(this.state().sortedValues);
+		this.onTouched();
+	}
+
+	sliderClick($event: any) {
+		this._widget.actions.updateSliderSize();
+		this.widget.actions.click(this.state().vertical ? $event.clientY : $event.clientX);
```

Comment:
I think you are doing here what should be done in the core.
Remove the `sliderClick` method here, and just call directly `widget.actions.click($event)` in the template.

Reply:
agree at this point we don't need the additional handling outside the core